src/OrchardCore.Modules/OrchardCore.Admin/Admin/AdminSettings.cs

```diff
@@ -1,7 +1,56 @@
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
 namespace OrchardCore.Admin.Models
 {
     public class AdminSettings
     {
         public bool DisplayMenuFilter { get; set; }
+        public string BrandImageUrl { get; set; }
+        public string FaviconUrl { get; set; }
+        public string Head { get; set; }
+
+        public string BrandImage
+        {
+            get
+            {
+                string path = null;
+
+                if (!string.IsNullOrEmpty(BrandImageUrl))
+                {
+                    var media = JArray.Parse(BrandImageUrl);
```

Comment:
I don't like the fact that you parse the BrandImageUrl here and make use of 2 different string vars to return the proper unique value from the json set in BrandImageUrl. You should parse this JSON directly in the Driver instead.